packages/paste-website/src/components/shortcodes/component-header/index.tsx

```diff
@@ -122,10 +122,12 @@ const ComponentHeader: React.FC<ComponentHeaderProps> = ({
           </PackageValue>
         </Box>
         <Box marginBottom="space20">
-          <PackageLabel>Install</PackageLabel>
+          <PackageLabel>Use from</PackageLabel>
```

Comment:
```suggestion
          <PackageLabel>Usage</PackageLabel>
```

Reply:
If you want to go all in, just add package name as an extra field and then just import

![image](https://user-images.githubusercontent.com/368249/98322124-8bc0ea80-1f9b-11eb-8ebd-289240dd24a6.png)

Reply:
Would this be confusing? That's two different packages 🤔

Reply:
Not sure if it will be confusing or not. Maybe make that change and see if we get any feedback from users.

Reply:
I kind of did the middle ground, let me know what yall think:

![image](https://user-images.githubusercontent.com/1592327/98611564-22551a80-22c0-11eb-8449-be50253f1a24.png)